### PR TITLE
Add script to help with linking sdk

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

```
cd audius-client
npm run sdk:link
npm run web:stage # or similar
```

@piazzatron 's idea

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/package.json [hunk 1]
```diff
@@ -7,12 +7,12 @@
   "private": true,
   "scripts": {
     "publish-scripts": "./scripts/publishScripts.sh",
-    "start": "craco --max-old-space-size=8076 start",
+    "start": "craco --max-old-space-size=12288 start",
```
Comment (sliptype) [inline packages/web/package.json:10]: Why this bump?

**Review by sliptype — APPROVED** [review]
Amazing idea 🔥

File: packages/web/package.json (hunk 1 above)
Comment (raymondjacobson) [reply packages/web/package.json:10]: idk! i oomed today randomly. :( maybe not needed?

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-link-sdk

**Review by piazzatron — APPROVED** [review]
amazing tysm

**Comment by piazzatron** [comment]
hmm... it seems to be indefinitely bundling for me 👀

**Comment by raymondjacobson** [comment]
> hmm... it seems to be indefinitely bundling for me 👀

can you may more? for me it finishes in a state where i have bundles for everything. there are _several_ bundles to produce tho

**Comment by raymondjacobson** [comment]
> > hmm... it seems to be indefinitely bundling for me 👀
> 
> can you say more? for me it finishes in a state where i have bundles for everything. there are _several_ bundles to produce tho

tested this again, it works... can FUP if there's an issue

**Pull request merged**